packages/react-examples/src/react-provider/CSSBaseline/CSSBaseline.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import { FluentProvider, CSSBaseline } from '@fluentui/react-provider';
+import { teamsLightTheme } from '@fluentui/react-theme';
+import * as React from 'react';
+
+const containerStyles = {
+  border: `5px solid var(--neutralColorTokens-neutralStroke1)`,
+  backgroundColor: 'var(--neutralColorTokens-neutralBackground1)',
+  color: 'var(--neutralColorTokens-brandForeground)',
+
+  margin: '5px',
+  padding: '5px',
+};
+
+export const Base = () => (
+  <>
+    <CSSBaseline />
+    <p>This example shows usage of "dir" attribute to change text direction.</p>
```

Comment:
I don't think that a description is relevant for this example, can you please update it?